readme.md

```diff
@@ -60,10 +71,12 @@ The instance is an [`Iterable`](https://developer.mozilla.org/en/docs/Web/JavaSc
 
 Both `key` and `value` can be of any type.
 
-#### .set(key, value)
+#### .set(key, value, maxAge)
 
 Set an item. Returns the instance.
 
+Individual expiration of an item can be specified with the `maxAge` parameter. If not specified, the global `maxAge` value will be used in case that is specified on the constructor, else the item will not have expiration.
```

Comment:
```suggestion
Individual expiration of an item can be specified with the `maxAge` parameter. If not specified, the global `maxAge` value will be used in case it is specified on the constructor, otherwise the item will never expire.
```